src/quilla/__init__.py

```diff
@@ -31,7 +33,7 @@ def make_parser() -> argparse.ArgumentParser:  # pragma: no cover
         Program to provide a report of UI validations given a json representation
         of the validations or given the filename containing a json document describing
         the validations
-        '''
+        ''',
```

Comment:
extra comma

Reply:
That one was added intentionally since the other function call lines for the parser also have a trailing comma. This is so if/when new lines need to be added to the parser setup it doesn't also register this line as having been changed